.github/workflows/check-filebeat.yml

```diff
@@ -1,32 +0,0 @@
-name: check-filebeat
-
-on:
-  pull_request:
-    paths:
-      - '.github/workflows/check-filebeat.yml'
-      - 'filebeat/**'
-      - 'x-pack/filebeat/**'
```

Comment:
This workflow is triggered for `filebeat` and `x-pack/filebeat` changes. 

So the exact changes you did in the `filebeat-pipeline.yml` should also be replicated on the x-pack flavour of it `.buildkite/x-pack/pipeline.xpack.filebeat.yml`